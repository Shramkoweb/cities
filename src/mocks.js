export const OFFERS = [
  {
    placeName: `Beautiful & luxurious apartment at great location`,
    img: `img/apartment-01.jpg`,
    price: 120,
    rating: 43,
    type: `Apartment`
  },
  {
    placeName: `Wood and stone place`,
    img: `img/apartment-02.jpg`,
    price: 80,
    rating: 12,
    type: `Private room`
  },
  {
    placeName: `Canal View Prinsengracht`,
    img: `img/apartment-03.jpg`,
    price: 132,
    rating: 66,
    type: `Apartment`
  },
  {
    placeName: `Nice, cozy, warm big bed apartment`,
    img: `img/apartment-02.jpg`,
    price: 180,
    rating: 100,
    type: `Apartment`
  },
];
