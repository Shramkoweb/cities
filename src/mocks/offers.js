export const OFFERS = [
  {
    id: 834576,
    city: `Amsterdam`,
    title: `Beautiful & luxurious apartment at great location`,
    previewPhoto: `img/apartment-01.jpg`,
    isPremium: true,
    isFavorite: true,
    price: 120,
    rating: 43,
    type: `Apartment`,
    coordinates: [52.3909553943508, 4.85309666406198]
  },
  {
    id: 786198,
    city: `Amsterdam`,
    title: `Wood and stone place`,
    previewPhoto: `img/apartment-02.jpg`,
    price: 80,
    isPremium: true,
    isFavorite: false,
    rating: 12,
    type: `Private room`,
    coordinates: [52.369553943508, 4.85309666406198]
  },
  {
    id: 123072,
    city: `Amsterdam`,
    title: `Canal View Prinsengracht`,
    previewPhoto: `img/apartment-03.jpg`,
    price: 132,
    isPremium: false,
    isFavorite: true,
    rating: 66,
    type: `Bungalow`,
    coordinates: [52.3909553943508, 4.929309666406198]
  },
  {
    id: 788764,
    city: `Amsterdam`,
    title: `Nice, cozy, warm big bed apartment`,
    previewPhoto: `img/room.jpg`,
    price: 180,
    isPremium: false,
    isFavorite: false,
    rating: 100,
    type: `Apartment`,
    coordinates: [52.3809553943508, 4.939309666406198]
  },
];
