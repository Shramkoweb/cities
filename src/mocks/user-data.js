export const USER_DATA = `serhii.shramko@gmail.com`;
