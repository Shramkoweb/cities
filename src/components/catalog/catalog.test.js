import React from "react";
import renderer from "react-test-renderer";
import Catalog from "./catalog";
import {OFFERS} from "../../mocks/offers";

it(`Catalog component render correct`, () => {
  const catalog = renderer
    .create(<Catalog offers={OFFERS} />)
    .toJSON();
  expect(catalog).toMatchSnapshot();
});
