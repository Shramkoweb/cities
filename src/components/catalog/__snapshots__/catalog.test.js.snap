// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Catalog component render correct 1`] = `
<main
  className="page__main page__main--index"
>
  <h1
    className="visually-hidden"
  >
    Cities
  </h1>
  <div>
    Active: 
  </div>
  <div
    className="cities"
  >
    <div
      className="cities__places-container container"
    >
      <section
        className="cities__places places"
      >
        <h2
          className="visually-hidden"
        >
          Places
        </h2>
        <b
          className="places__found"
        >
          10
           places to stay in 
          NY
        </b>
        <div>
          Active: 
        </div>
      </section>
      <div
        className="cities__right-section"
      />
    </div>
  </div>
</main>
`;
